Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -1102,6 +1102,17 @@ + (NSData *)getClientDataJSONForAuthenticationType:(_WKWebAuthenticationType)typ
     return clientDataJSON.autorelease();
 }
 
++ (NSData *)getClientDataJSONWithTopOrigin:(_WKWebAuthenticationType)type challenge:(NSData *)challenge origin:(NSString *)origin topOrigin:(NSString *)topOrigin crossOrigin:(WebAuthn::Scope)crossOrigin
```

Comment:
Where are the header changes for this?

Also, we likely want to keep using a `BOOL crossOrigin` in the API since we cannot expose `WebAuthn::Scope` is an internal type that cannot be used in API.